src/components/AxisCollection/index.js

```diff
@@ -83,27 +89,22 @@ class AxisCollection extends React.Component {
           height={height}
           offsetx={numVisible * yAxisWidth}
           width={yAxisWidth}
+          onMouseEnter={this.onAxisMouseEnter('collapsed')}
+          onMouseLeave={this.onAxisMouseLeave('collapsed')}
         />
       );
     }
     return null;
   }
 
   render() {
-    const { width, height, onMouseEnter, onMouseLeave } = this.props;
+    const { width, height } = this.props;
 
     // We need to render all of the axes (even if they're hidden) in order to
     // keep the zoom states in sync across show/hide toggles.
-    const axes = this.renderAllAxes();
+    const axes = this.renderAllVisibleAxes();
     return (
-      <svg
-        width={width}
-        height={height}
-        onMouseEnter={onMouseEnter}
-        onMouseLeave={onMouseLeave}
-        onFocus={onMouseEnter}
-        onBlur={onMouseLeave}
```

Comment:
Just curious, why do you think we don't need those `onFocus` and `onBlur` events to trigger?